log4j-1.2-api/src/test/java/org/apache/log4j/config/PropertiesConfigurationTest.java

```diff
@@ -129,6 +129,28 @@ public void testConsoleAppenderLevelRangeFilter() throws Exception {
             assertEquals(Level.ALL, customFilterReal.getMinLevel());
             final LevelRangeFilter defaultFilter = (LevelRangeFilter) filters[1];
             assertEquals(Level.TRACE, defaultFilter.getMinLevel());
+
+            final ListAppender legacyAppender = (ListAppender) ((AppenderAdapter.Adapter) appender).getAppender();
+            final Logger logger = LogManager.getLogger(PropertiesConfigurationTest.class);
+
+            // denied???
+            logger.trace("TRACE");
+            assertEquals(0, legacyAppender.getEvents().size());
+            // denied???
+            logger.debug("DEBUG");
+            assertEquals(0, legacyAppender.getEvents().size());
+            // denied???
+            logger.info("INFO");
+            assertEquals(0, legacyAppender.getEvents().size());
+            // denied???
+            logger.warn("WARN");
+            assertEquals(0, legacyAppender.getEvents().size());
+            // denied???
+            logger.error("ERROR");
+            assertEquals(0, legacyAppender.getEvents().size());
+            // denied???
+            logger.fatal("FATAL");
+            assertEquals(0, legacyAppender.getEvents().size());
```

Comment:
this test fails in ba48d2b, so I took a look here.
Under this setting, this rejects all output logs.

I believe we expect here we get all messages accepted because we set `levelMin=ALL`.
(see: https://issues.apache.org/jira/browse/LOG4J2-2315)